sheets/lvcreate

```diff
@@ -6,13 +6,16 @@
 # Create a logical volume of 10 gigabytes in the volume group vg1:
 lvcreate -L 10G vg1
 
-# Create a 1500 megabyte linear logical volume named mylv in the volume group vg1:
+# Create a 1500 megabyte linear logical volume named mylv
```

Comment:
They said "megabyte", so using MB coincides with that. The number "1500" can be shortened to 1.5MB without any problems, and allows you to remove "megabyte".